.github/workflows/release.yaml

```diff
@@ -0,0 +1,40 @@
+name: Release Workflow
+on:
+  workflow_dispatch:
+    inputs:
+      release-version:
+        description: 'Version being released'
+        required: true
+      branch:
```

Comment:
Is the `branch` input used automatically or should it be referenced in the checkout step?

Reply:
default branch name is main in line 11. unless managing multiple releases in a different branch it shouldn't be changed.

Reply:
Sure, but I'm not seeing the `github.event.inputs.branch` referenced anywhere so was curious how it's used

Reply:
Actually the triggered workflow receives the inputs in the [github.event](https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context) context.